### PR TITLE
Dockerfile and docker-compose for local dev (postgres only)

### PR DESCRIPTION
**Pull request by steadyember**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+version: "3.5"
+services:
+  db:
+    image: postgres
+    environment:
+      POSTGRES_USER: throat
+      POSTGRES_PASSWORD: dev
+      POSTGRES_DATABASE: throat
```
Comment (Polsaker) [inline docker-compose.yml:8]:
```suggestion
      POSTGRES_DATABASE: throat
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U postgres"]
      interval: 5s
      timeout: 5s
      retries: 5
```

Comment (steadyember) [reply docker-compose.yml:8]: Got it, this looks reasonable. I will get that incorporated.

Comment (steadyember) [reply docker-compose.yml:8]: Arg. This made it fail for me locally where it was working before :(

Comment (steadyember) [reply docker-compose.yml:8]: https://stackoverflow.com/questions/31746182/docker-compose-wait-for-container-x-before-starting-y/41854997#41854997

Comment (steadyember) [reply docker-compose.yml:8]: https://docs.docker.com/compose/startup-order/

Comment (steadyember) [reply docker-compose.yml:8]: I added an additional command to the migration service for `restart:on_failure` this is working locally for me and I also kept the healthcheck in.

**Pull request merged**